### ISSUE TITLE
NEW IDEA FOR TIBIALYZER!

### ISSUE THREAD

**Issue by jadap93** [Mytherin/Tibialyzer#310]
hello my brother first of all i want to say ur a god by doing this program, so good and most important completely legl for true players like us, i just want to sahre an idea, right now with the health HUD i dont see my equipment anymore too look at my HP, but when we are in war wearing up might rings and stone skins its pretty hard to see where are you running and the equipment you need to wear up so i mean would be great if you can create an HUD just for necklaces and rings just like HP HUD where you can position it where you want, necklace and ring HUD would be great so you can see at your HP HUD while you see your ring and necklace HUD, good for war and also good for hunts like banuta where you need to wear up everytime garlic neclakces so would be easier to see where its broken the garlic, i hope you do something for us !!!

**Comment by dbbj** [comment]
I don't think that can be done, based on what I've seen, tibialyzer has to get the text information somewhere, be it your health/mana/exp numbers or the server log and loot channel messages.
Unless you keep looking at your rings and amulets, there is no way of tibialyzer getting that info.
